tests/unit/s2n_encrypted_extensions_test.c

```diff
@@ -58,23 +71,91 @@ int main(int argc, char **argv)
         EXPECT_SUCCESS(s2n_connection_free(server_conn));
     }
 
-    /* Client successfully receives empty encrypted extension */
+    /* Test that server sends Encrypted Extensions  */
+    {
+        const uint8_t ENCRYPTED_EXTENSIONS_HEADER_SIZE = 2;
+        struct s2n_connection *server_conn;
+        EXPECT_NOT_NULL(server_conn = s2n_connection_new(S2N_CLIENT));
+        EXPECT_SUCCESS(s2n_connection_set_config(server_conn, config));
```

Comment:
are trying to create a client or sever connection ?

Reply:
good catch, the behavior doesn't change here, but i should update this to be consistent.